package.json

```diff
@@ -36,6 +36,7 @@
     "cmdk": "^0.2.1",
     "driver.js": "^1.3.1",
     "framer-motion": "^11.0.13",
+    "lodash": "^4.17.21",
```

Comment:
this should be removed as well and furthermore regenerate the yarn.lock without these dependencies otherwise you will conflict with main